### ISSUE TITLE
Question Regarding Training Setup for Diffusion Model

### ISSUE THREAD

**Issue by mathjihun** [alexdenker/SteerableConditionalDiffusion#4]
Hi, thank you for sharing your excellent work.

I was wondering if you could share some details about the training setup you used when training the diffusion model on elliptical data. I followed the approach from guided-diffusion [[link](https://github.com/openai/guided-diffusion)], and used the parameter values in configs/diskellipses.yml along with the defaults from the original repository. Would this be the correct setup?

In particular, I’m curious about the batch size — did you use 256 during training?

Best regards,
Jihun

**Comment by alexdenker** [comment]
Hello Jihun, 

Yes we used the standard DDPM training setup. The 256 in the configs file refers to the image dimension. 
For training, we used a batch size of 4, learning rate 1e-4 and made use of EMA. I can upload a training script to the repository. 

Best,
Alex

**Comment by mathjihun** [comment]
Thank you so much for your help.
Would it be possible for you to upload the training script to the repository when you have a chance? That would be greatly appreciated.

**Comment by alexdenker** [comment]
No problem, I just pushed an example training script which should work: https://github.com/alexdenker/SteerableConditionalDiffusion/blob/main/train_uncond_diskellipses.py

The default training duration is 100 epochs (about 1Mio gradient updates). However, even with 100K (about 10 epochs) the unconditional model should work.

**Comment by mathjihun** [comment]
Thank you very much for your help. It was incredibly helpful.

**Comment by alexdenker** [comment]
If you come across any bugs or something is not clear, feel free to let me know!
